### DISCUSSION TITLE
Understanding Force Application: Exploring 'n_forced_lsps' and 'prctForced' Variables"

### DISCUSSION THREAD

**Discussion by rishabhbhatt233** [UvaCsl/HemoCell#119] [General discussion]
Hello,
I'm currently diving into the topic of force applied per unit area and exploring how to determine the specific area where the force is applied within a codebase. In this context, I'm curious about how the variables "n_forced_lsps" in stretchcell.cpp and "prctForced" in config.xml are works.

Thanks

**Comment by gzavo** [comment]
Hi @rishabhbhatt233 ,

As I recall, 'prctForced' is the percentage of surface area being pulled. "n_forced_lsp" is the number of vertices this percentage translates to on the surface.
